spec/index.bs

```diff
@@ -12384,30 +12384,33 @@ attachments used by this encoder.
 
                 1. For each |bundle| in |bundles|:
                     1. Increment |this|.{{GPURenderCommandsMixin/[[drawCount]]}} by |bundle|.{{GPURenderBundle/[[drawCount]]}}.
+                    1. [$Enqueue a render command$] on |this| which issues the following steps on the
+                        [=Queue timeline=] with |renderState| when executed:
 
-                1. [=map/Clear=] |this|.{{GPUBindingCommandsMixin/[[bind_groups]]}}.
-                1. Set |this|.{{GPURenderCommandsMixin/[[pipeline]]}} to `null`.
-                1. Set |this|.{{GPURenderCommandsMixin/[[index_buffer]]}} to `null`.
-                1. [=map/Clear=] |this|.{{GPURenderCommandsMixin/[[vertex_buffers]]}}.
+                        <div data-timeline=queue>
+                            [=Queue timeline=] steps:
 
-                1. Let |passState| be a snapshot of |this|'s current state.
-                1. [$Enqueue a render command$] on |this| which issues the subsequent steps on the
-                    [=Queue timeline=] with |renderState| when executed.
-            </div>
-            <div data-timeline=queue>
-                [=Queue timeline=] steps:
+                            1. Execute each command in |bundle|.{{GPURenderBundle/[[command_list]]}}
+                                with |renderState|.
 
-                1. For each |bundle| in |bundles|:
-                    1. Execute each command in |bundle|.{{GPURenderBundle/[[command_list]]}}
-                        with |passState| and |renderState|.
+                                Note: |renderState| cannot be changed by executing render bundles.
```

Comment:
Add a note, maybe to this note, that bindingState was already captured at encoding time, so it isn't used by the bundle commands.